src/librustc_trait_selection/traits/object_safety.rs

```diff
@@ -170,6 +170,24 @@ fn object_safety_violations_for_trait(
     violations
 }
 
+fn trait_bound_spans<'tcx>(
```

Comment:
`sized_trait_bound_spans`, maybe?

Reply:
Yea I wasn't really satisfied with the current name, yours is better.